src/main/java/io/ebean/config/properties/Loader.java

```diff
@@ -112,59 +112,42 @@ void loadWithExtensionCheck(String fileName) {
    * Evaluate all the configuration entries and return as properties.
    */
   Properties eval() {
-		return loadContext.eval();
-	}
+    return loadContext.eval();
+  }
 
-	void loadYaml(String resourcePath, Source source) {
-		if (yamlLoader != null) {
-		  try {
-			  yamlLoader.load(resource(resourcePath, source));
+  void loadYaml(String resourcePath, Source source) {
+    if (yamlLoader != null) {
+      try (InputStream is = resource(resourcePath, source)) {
+        yamlLoader.load(is);
       } catch (Exception e) {
         log.warn("Failed to read yml from:" + resourcePath, e);
```

Comment:
Uses try with resources now.
@rbygrave  should we throw an exception instead of printing a warning if the yml file could not be read?